rust_build_utils/rust_utils.py

```diff
@@ -415,8 +434,12 @@ def post_build(project: Project, config: CargoConfig, packages: PackageList) ->
 
 def run_command(command):
     print("|EXECUTE| {}".format(" ".join(command)))
-    subprocess.check_call(command)
-    print("")
+    p = subprocess.run(command, capture_output=True, text=True)
```

Comment:
I don't agree with this change.
- This breaks log streaming. If the build takes 5 minutes, you won't see any logs until the build has finished. This also breaks output colouring when used interactively.
- Command output is very useful when debugging CICD. Only printing the output for commands that failed would make it more difficult to understand what is happening under the hood.

Reply:
Yep, was added for debugging and done so incorrectly. Will remove.